src/SourceBuild/content/eng/tools/BinaryToolKit/BinaryToolCli/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using BinaryTool;
+
+namespace BinaryToolCli;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        if (args.Length < 2 || args.Length > 6)
```

Comment:
It's fine to do this manually. Just wanted to mention that we usually use System.CommandLine for this.